### DISCUSSION TITLE
is my BM280 with problem?

### DISCUSSION THREAD

**Discussion by Aripaule** [aaronwmorris/indi-allsky#1627] [Q&A] [answered]
hi,

Firstly, I would like to thank the creators for the project. 

I have a possible problem with my bm280 sensor, as it is showing a pressure of around 900hpa, when it should be around 1000, as we can confirm from the data on the openweather website.

I would also like to know where the acquired data from openweather is shown in the interface (I didn't find it in the charts tab).

thank you very much!
![Imagem do WhatsApp de 2024-11-09 à(s) 22 20 08_8590125f](https://github.com/user-attachments/assets/0a4dd16f-349b-497c-b758-615aad5a5521)

**Comment by aaronwmorris** [comment]
Interesting.  What is your elevation?

On the Admin tab of the config, you can configure the 4 custom charts to be the various sensor values.

**Reply by Aripaule** [reply]
my elevation is 1000 m above sea level.

and,


I found where to enable the graphics, are we limited to just 4 slots for now?

**Reply by aaronwmorris** [reply, marked as answer]
I have been reading that weather services will normalize the pressure measurements to sea level which is probably why you are seeing a difference.  With an elevation of 1000m @ 15c, the pressure should be ~898 hPa which is pretty close to your measurement.

For now, it is limited to 4 custom graph slots.

**Comment by Aripaule** [comment]
hi @aaronwmorris,
as you said in "Sensors", mine is not showed with command i2cdetect too,
and also it seems to be connected in 0x29 like TSL 2591.
![Imagem do WhatsApp de 2024-11-23 à(s) 12 47 37_1631aa68](https://github.com/user-attachments/assets/3438a559-cba9-472e-aefb-775fa3746ccf)
![Imagem do WhatsApp de 2024-11-23 à(s) 12 47 51_47cecdca](https://github.com/user-attachments/assets/f8af5d15-521c-4163-a376-4f9f604bbdf1)


Could this cause any problems?

**Reply by aaronwmorris** [reply]
The problem appears to be your MLX90614.  I would double check your wiring to make sure it is connected properly.

**Reply by Aripaule** [reply]
it worked! I redid the DUpont connections and the sensor appeared correctly.

Furthermore, do you know the unit of measurement for this SQM tab? is it mag/arsec²?

![image](https://github.com/user-attachments/assets/cc9e270f-4ec1-4494-b04c-e668678af57f)

**Reply by aaronwmorris** [reply]
It is supposed to be mag/arcsec², but there probably needs to be more work done in this calculation.
